### PR TITLE
218 vertimas į anglų kalbą

### LINKED ISSUE #218: Vertimas į anglų kalbą
Visi svetainės tekstai išversti į anglų kalbą.

---

Sugeneruoti kalbų failus (`.po`) ir juos išversti. Į dokumentaciją įtraukti informaciją, kaip iš `.po` failų generuoti `.mo` failus.

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#localization-how-to-create-language-files

Sutvarkyti lietuvių kalbos formatus, kad datos ir laikas būtų atvaizduojami lietuvišku formatu (YYYY-MM-DD ir HH:MM). Tačiau jei parinkta anglų kalba tada rodyti anglišką formatą.

https://docs.djangoproject.com/en/3.2/topics/i18n/formatting/

https://poedit.net/

### PR DESCRIPTION
**Pull request by adp-atea**

### CODE REVIEW

File: locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,3320 @@
...
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: .\venv\Lib\site-packages\_pytest\config\argparsing.py:458
+#, python-format
+msgid "ambiguous option: %(option)s could match %(matches)s"
```
Comment (sirex) [inline locale/en/LC_MESSAGES/django.po]: We need only messages, from `vitrina` package, not from all other packages.

**Review by sirex — CHANGES_REQUESTED** [review]

File: vitrina/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,1702 @@
...
+#: vitrina/datasets/templates/vitrina/datasets/coretable.html:53
+msgid ""
+"Duomenų rinkinio atvėrimo būsena: Inventorintas, parengti metaduomenys, "
+"planuojamas atverti, atvertas, nenumatytas atverti"
+msgstr ""
+"Dataset status: inventoryed, prepared metadata, planned to open, opened not "
+"schedules to open"
+
+#: vitrina/datasets/templates/vitrina/datasets/coretable.html:64
+msgid "dataset.status"
```
Comment (sirex) [inline vitrina/locale/en/LC_MESSAGES/django.po]: Please also fix these messages, by replacing them into a human readable lithuanian text and translating it to english.

**Review by sirex — CHANGES_REQUESTED** [review]
Looks good, but it would be good to fix messages like `dataset.status` by replacing to natural lithuanian language. This will probably require opening page were a message was used in order to understand what message is supposed to be shown.

File: vitrina/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (adp-atea) [reply vitrina/locale/en/LC_MESSAGES/django.po]: Fixed instances like this that I could find, but for example harvested value inside coretable.html needs to have some functionality tied to it, for now added it as an empty string and wrote a TODO for it.

**Review by sirex — APPROVED** [review]

**Pull request merged**